corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py

```diff
@@ -171,15 +175,13 @@ def ensure_plans(dry_run, verbose, for_tests, apps):
                     if hasattr(SoftwarePlanVersion, 'product_rates'):
                         software_plan_version.save()
 
-                    for product_rate in product_rates:
-                        product_rate.save()
-                        if hasattr(SoftwarePlanVersion, 'product_rates'):
-                            software_plan_version.product_rates.add(product_rate)
-                        elif hasattr(SoftwarePlanVersion, 'product_rate'):
-                            assert len(product_rates) == 1
-                            software_plan_version.product_rate = product_rate
-                        else:
-                            raise AccountingError('SoftwarePlanVersion does not have product_rate or product_rates field')
+                    product_rate.save()
+                    if hasattr(SoftwarePlanVersion, 'product_rates'):
```

Comment:
I know this isn't part of the changes here, but out of curiosity, when would a `SoftwarePlanVersion` have a `product_rates` attribute? I couldn't find it anywhere in the model, and it doesn't seem like we have a junction table to store a many-to-many relationship.